system/jlib/jdebug.cpp

```diff
@@ -3075,7 +3075,7 @@ void printProcMap(const char *fn, bool printbody, bool printsummary, StringBuffe
                 else if (strcmp(perms,"r--p")==0)
                         t = segtype_const;
                 else {
-                    ERRLOG("%s - unknown perms",perms);
+                    OERRLOG("%s - unknown perms",perms);
```

Comment:
probably IERR.  This is unexpected permissions reported by the operating system, not something the operator can change.